src/client/public/index.html

```diff
@@ -24,6 +24,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <!-- Bootstrap-->
+    <link
```

Comment:
A nicer way to do this (since we're running with NPM) is to install the bootstrap package and then include that. https://getbootstrap.com/docs/5.0/getting-started/download/#npm

...but let's do that later.